### ISSUE TITLE
[Feature Request]: Classical GLMM

### ISSUE THREAD

**Issue by cfrowd1** [jasp-stats/jasp-issues#1583]
### Description

Making contrasts easier to specify

### Purpose

More convenient--would no doubt help uaers who don't know about the sum averaging procedure.

### Is your feature request related to a problem?

No

### Describe the solution you would like

See above

### Additional context

Perhaps also consider including different methods for specifying the Covaiance Structure.

**Comment by cfrowd1** [comment]
Yes, that's right, Julius, a simpler method of specifying contrasts.  It can be difficult for some user to understand how these work (the patterns of +/- 1's and 0's) and a more straightforward way would be valuable.  With best regards, Charlie.

--
Charlie Frowd CPsychol CSci AFBPsS
Professor of Forensic Psychology, School of Psychology and Computer Science, University of Central Lancashire, Preston PR1 2HE.
#EvoFIT >> www.evofit.co.uk
________________________________

**Comment by tomtomme** [comment]
@cfrowd1 
Do you have an idea on how to make it simpler? Any other software that does it better?

Maybe users could just click on the variables to contrast.
They even could shift+click on multiple ones to group them. 
But if one would want to apply different weights for different vars, then this would still force the user to input a numerical value somewhere. And then we would be back, where we started from.

**Comment by tomtomme** [comment]
@cfrowd1 
![grafik](https://github.com/jasp-stats/jasp-issues/assets/4428185/4ea15bb7-89f9-4c97-b644-8b2c23837f43)

This does not seem to include slopes however...

**Comment by cfrowd1** [comment]
I thought that slopes could be included: there's a section to mention them, unless I've misunderstood. Any decent GLMM would want them to be there... With best regards, Charlie.

--
Charlie Frowd CPsychol CSci AFBPsS
Professor of Forensic Psychology, School of Psychology and Humanities, University of Central Lancashire, Preston PR1 2HE. www.uclan.ac.uk/academics/charlie-frowd<https://www.uclan.ac.uk/academics/charlie-frowd>
EvoFIT >> www.evofit.co.uk<http://www.evofit.co.uk/>
EvoFIT on the 2022 Royal Institution Christmas Lectures: www.bbc.co.uk/iplayer/episode/m001gmxq/royal-institution-christmas-lectures-2022-dame-sue-black-3-living-body<https://www.bbc.co.uk/iplayer/episode/m001gmxq/royal-institution-christmas-lectures-2022-dame-sue-black-3-living-body>
________________________________

**Comment by cfrowd1** [comment]
Ah, that's in the latest model. Thank you!

Ok, for these contrasts then. It would be very helpful to have several options incl. at least simple contrasts.

For EMMEANS, could they calculate comparisons if an interaction is involved? At the moment, these are for only individual predictors.

With best regards, Charlie.

--
Charlie Frowd CPsychol CSci AFBPsS
Professor of Forensic Psychology, School of Psychology and Humanities, University of Central Lancashire, Preston PR1 2HE. www.uclan.ac.uk/academics/charlie-frowd<https://www.uclan.ac.uk/academics/charlie-frowd>
EvoFIT >> www.evofit.co.uk<http://www.evofit.co.uk/>
EvoFIT on the 2022 Royal Institution Christmas Lectures: www.bbc.co.uk/iplayer/episode/m001gmxq/royal-institution-christmas-lectures-2022-dame-sue-black-3-living-body<https://www.bbc.co.uk/iplayer/episode/m001gmxq/royal-institution-christmas-lectures-2022-dame-sue-black-3-living-body>
________________________________

**Comment by FBartos** [comment]
Adding the "Help" button with a simple explanation in English is a good idea! I will check how to add that.

With respect to interactions, you specify contrasts on interactions using the +/- coding as you would do with main effects. The example in data library explains this using the Lark and Owls data set:
![image](https://github.com/jasp-stats/jasp-issues/assets/38475991/be368480-3e00-4b8b-b630-07a5ee28513e)

**Comment by cfrowd1** [comment]
Thank you.  The problem with +/- coding is that it is complicated and people don't really understand it.  Pushing a button that can do the contrasts for you would be ideal.  This would make the software much more accessible.  More of my students, for example, would be able to use it.  Do the coding for the user...  Just a thought.  With best regards, Charlie.

--
Charlie Frowd CPsychol CSci AFBPsS
Professor of Forensic Psychology, School of Psychology and Humanities, University of Central Lancashire, Preston PR1 2HE. www.uclan.ac.uk/academics/charlie-frowd<https://www.uclan.ac.uk/academics/charlie-frowd>
EvoFIT >> www.evofit.co.uk<http://www.evofit.co.uk/>
EvoFIT on the 2022 Royal Institution Christmas Lectures: www.bbc.co.uk/iplayer/episode/m001gmxq/royal-institution-christmas-lectures-2022-dame-sue-black-3-living-body<https://www.bbc.co.uk/iplayer/episode/m001gmxq/royal-institution-christmas-lectures-2022-dame-sue-black-3-living-body>
________________________________
